server/src/migrations/20210510160632_addCustomerModel.ts

```diff
@@ -10,6 +10,7 @@ export async function up(knex: Knex): Promise<any> {
         .inTable('roadmaps')
         .onDelete('CASCADE');
       table.string('name', 75).notNullable();
+      table.string('email', 75).nullable();
```

Comment:
```suggestion
      table.string('email', 255).nullable();
```

Tämä olisi hyvä laitaa vähintäänkin yhtä sallivaksi kuin json schema, muutoin samalle virheelle on useampi lähde.